app/Filament/Pages/Settings/NotificationPage.php

```diff
@@ -214,21 +214,30 @@ public function sendTestTelegramNotification()
     {
         $notificationSettings = new (NotificationSettings::class);
 
-        if (count($notificationSettings->telegram_recipients)) {
-            foreach ($notificationSettings->telegram_recipients as $recipient) {
-                \Illuminate\Support\Facades\Notification::route('telegram_chat_id', $recipient['telegram_chat_id'])
-                ->notify(new TelegramNotification('Test notification channel *telegram*'));
+        $bot = \Config::get('telegram.bot');
```

Comment:
So the best way to reference config values is through the helper method `config()`, in this case if you want to call an item in the `bot` array the syntax is `config('filename.array.key`) or `config('telegram.bot.token')`.